bpf/flow.h

```diff
@@ -60,7 +62,6 @@ typedef struct flow_id_t {
     struct in6_addr src_ip; // keep these aligned
     struct in6_addr dst_ip;
     u16 eth_protocol;
-    u8 direction;
```

Comment:
direction field moved from `flow_id` to `flow_metrics`, because:
- when the calculation is based on TCP flags, it can be different in the next packets around the same flow
- it is unnecessary in flow_id. Src and dst parameters are good enough